core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -77,13 +78,23 @@ public MerlinBuilder withAllCallbacks() {
     }
 
     /**
-     * Sets the internal Merlin Logging state. Uses the TAG : Merlin
+     * Sets the internal Merlin logging state to report to a given LogHandle.
      *
-     * @param withLogging by default Logging is disabled. withLogging = true will enable logging, withLogging = false will disable.
+     * @param logHandle that should be used when logging.
      * @return MerlinBuilder.
      */
-    public MerlinBuilder withLogging(boolean withLogging) {
-        MerlinLog.LOGGING = withLogging;
+    public MerlinBuilder withLogging(Logger.LogHandle logHandle) {
```

Comment:
Isn't this representing an API breaking change? Users upgrading to this version of the library might have to fix and recompile. That's annoying. 

Can we instead deprecate the method that we previously had and make the `true` perhaps attach a default `logHandle` and the false do nothing?